index.html

```diff
@@ -429,6 +429,21 @@ <h3 class="card__title">Comments</h3>
               </div>
             </div>
           </label>
+          <label>
+            <input id="experienceUrl" name="experienceUrl" type="radio" value="croupier" class="branded-content-card" />
+            <div class="card">
+              <picture>
+                <source media="(max-width: 425px)" srcset="./assets/cards/card-homepage-425.png" />
+                <img class="card__image" src="./assets/cards/card-homepage-425w.png" />
+              </picture>
+              <div class="card__content">
+                <h3 class="card__title">Branded Content</h3>
+                <p class="card__text">
+                  Flowcard served from AdManager.
```

Comment:
Technically, they are not served from AdManager, maybe we'd need help from Marketing team here.

```suggestion
                  Define a branded content article from AdManager and load it in a Flowcard.
```